src/main/java/boot/bootdemo/service/ReviewService.java

```diff
@@ -0,0 +1,8 @@
+package boot.bootdemo.service;
+
+import boot.bootdemo.model.Review;
+import java.util.List;
+
+public interface ReviewService {
+    void saveAll(List<Review> entities);
```

Comment:
let's also add UserService and ProductService with `saveAll()` method

and also let's implement saving all the data that we read from the file in this PR